Apps/WebClient/src/ClientApp/src/views/landing.vue

```diff
@@ -346,30 +351,52 @@ interface Tile {
     },
 })
 export default class LandingView extends Vue {
+    private logo: string = Image00;
+    private devices: string = Image02;
+    private bottomImage: string = Image07;
+    private isOpenRegistration: boolean = false;
+    private getTileClass(index: number): string {
+        return index % 2 == 0 ? "order-md-1" : "order-md-2";
+    }
+    @Getter("webClient", { namespace: "config" })
+    webClientConfig!: WebClientConfiguration;
+
+    private mounted() {
+        this.isOpenRegistration =
+            this.webClientConfig.registrationStatus == RegistrationStatus.Open;
+        this.icons[0].active = this.webClientConfig.modules[
+            "MedicationHistory"
+        ];
+        this.icons[1].active = this.webClientConfig.modules["Note"];
+        this.icons[2].active = this.webClientConfig.modules["Immunization"];
+        this.icons[3].active = this.webClientConfig.modules["Laboratory"];
+        this.icons[4].active = this.webClientConfig.modules["Encounter"];
+    }
+
     private icons: Icon[] = [
```

Comment:
nit, this is a variable definition, we usually have those above the method definition